### ISSUE TITLE
New feedback - ID 116

### ISSUE THREAD

**Issue by knudsvik** [equinor/MAD-VSM-WEB#326]
**User input**
Would like to visualise "Waiting" between to Main Activities

**Sharepoint list ID**
116

**User**
Elisabeth Siem

**Comment by knudsvik** [comment]
Will probably need to have fixed #172 before this, otherwise dropping a waiting card onto a main activity will place it underneath.

**Comment by SjoenH** [comment]
Yes, I'm looking into possible technical solutions for this in #296.
The underlying canvas-logic we have today is too simplistic for such interactions. :man_scientist:
